src/pytest_inmanta_lsm/remote_orchestrator.py

```diff
@@ -465,7 +465,10 @@ def remote_project_path(self) -> pathlib.Path:
         This path depends on the on-disk layout of the remote orchestrator.
         """
         if self._remote_project_path is None:
-            cmd = "if test -f /var/lib/inmanta/.inmanta_use_new_disk_layout ; then echo True ; fi"
+            cmd = (
+                "if test -f /var/lib/inmanta/.inmanta_use_new_disk_layout ||"
+                " -f /var/lib/inmanta/.inmanta_disk_layout_version; then echo True ; fi"
```

Comment:
Should we bother reading the actual version in the file and checking its value already?

Reply:
I don't think it does much at the moment. We wouldn't do anything with that value. @wouterdb what do you think?